sdk/servicebus/azure-servicebus/tests/async_tests/test_sessions_async.py

```diff
@@ -1167,4 +1167,24 @@ async def test_async_session_non_session_send_to_session_queue_should_fail(self,
             async with sb_client.get_queue_sender(servicebus_queue.name) as sender:
                 with pytest.raises(ServiceBusError):
                     message = ServiceBusMessage("Handler message")
-                    await sender.send_messages(message)
\ No newline at end of file
+                    await sender.send_messages(message)
+
+    @pytest.mark.asyncio
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedServiceBusResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    @ServiceBusQueuePreparer(name_prefix='servicebustest', requires_session=True)
+    @pytest.mark.parametrize("uamqp_transport", uamqp_transport_params, ids=uamqp_transport_ids)
+    @ArgPasserAsync()
+    async def test_async_session_by_session_client_conn_str_receive_handler_with_no_session(self, uamqp_transport, *, servicebus_namespace_connection_string=None, servicebus_queue=None, **kwargs):
```

Comment:
"...no_session" feels like it's not on a sessionful queue. Would calling this `test_async_session_by_session_client_conn_str_receive_handler_with_next_available_session` work?

Reply:
or name it similar to the sync test?